src/ameba/config.cr

```diff
@@ -1,5 +1,6 @@
 require "semantic_version"
 require "yaml"
+require "ecr/processor"
```

Comment:
Why this is added _here_?

Reply:
As this is where other requires for external things go and I tend to like having them all in the same place.